extensions/autoPepSIRF.py

```diff
@@ -254,6 +259,20 @@ def main():
             csD = io.fileDictFull(args.zscore, valType="float", rowNames=True)
             for r1, r2 in pD.items():
                 scatter(csD[r1], r1, csD[r2], r2, "zRepScatters/%s_%s_Z.%s" % (r1, r2, args.scatterFormat), plotLog=False)
+                
+        #Enriched peptides scaterplot generation
+        if args.enrichedScatters and args.colsum:
```

Comment:
At this level, let's add an "elif args.enrichedScatters:" statement with a print statement warning that the plots won't be generated because args.colsum is not set.